LocalSettings.php

```diff
@@ -2397,7 +2397,7 @@
 				'requestwiki' => true,
 			],
 			'wikicreator' => [
-				'createwiki' => true,
+				'read' => true,
```

Comment:
Can't we just comment this with `//` so it can be very easily uncommented later, rather than replacing it with read?

Reply:
> Can't we just comment this with `//` so it can be very easily uncommented later, rather than replacing it with read?

That's not a bad idea, actually. I like this.